.github/workflows/main.yaml

```diff
@@ -132,10 +132,7 @@ jobs:
         rustup default "$channel"
         rustup target add "${{ matrix.target }}"
 
-    # Our scripts use nextest if possible. This is skipped on the native ppc
-    # and s390x runners since install-action doesn't support them.
     - uses: taiki-e/install-action@nextest
-      if: "!(matrix.os == 'ubuntu-24.04-ppc64le' || matrix.os == 'ubuntu-24.04-s390x')"
```

Comment:
As I said in https://github.com/taiki-e/install-action/issues/1056#issuecomment-3111674746, it is actually probably better to use cache-cargo-install-action on these platforms.


> That said, considering that it is almost always necessary to do that on these platforms, using [cache-cargo-install-action](https://github.com/taiki-e/cache-cargo-install-action) is definitely better than install-action in terms of performance. (Eventually, the two may be merged, but there is no concrete plan yet: https://github.com/taiki-e/install-action/issues/66)